test/e2e/apimachinery/aggregator.go

```diff
@@ -72,13 +72,32 @@ var _ = SIGDescribe("Aggregator", func() {
 		aggrclient = f.AggregatorClient
 	})
 
-	It("Should be able to support the 1.10 Sample API Server using the current Aggregator", func() {
-		// Make sure the relevant provider supports Aggregator
+	// We have to put the version skew test and the storage version hash
+	// test in the same "It" block, because
```

Comment:
i don't follow, it looks to me that two tests are independent.

Reply:
Reworded.